src/engraving/dom/textbase.cpp

```diff
@@ -3370,11 +3392,36 @@ void TextBase::undoChangeProperty(Pid id, const PropertyValue& v, PropertyFlags
             undoChangeProperty(Pid::TEXT, stripText(false, false, true), propertyFlags(id));
         }
     }
-    if (id == Pid::FONT_STYLE || id == Pid::FONT_FACE || id == Pid::FONT_SIZE || id == Pid::TEXT_SCRIPT_ALIGN) {
-        // can't use standard change property as Undo might set to "undefined"
-        score()->undo(new ChangeTextProperties(m_cursor, id, v, ps));
-    } else {
+
+    bool isTextSpecificProperty = id == Pid::FONT_STYLE || id == Pid::FONT_FACE || id == Pid::FONT_SIZE || id == Pid::TEXT_SCRIPT_ALIGN;
```

Comment:
Here I see a lot more properties listed than in `textProperties()`. Of course, this is about something different, so the list need not be the same, but still... shouldn't these properties also be in `textProperties()`? Or otherwise, why only `FONT_STYLE` and these others not?